### PR TITLE
VRMLLoader: Fixed parsing bug when comments were present in the file.

### LINKED ISSUE #14837: VRMLLoader fails if there are comments in the file.
I have a VRML file that I have exported from Rhino, a CAD software. And it seems that rhino adds comments in the generated file, which results in an error when I try to import the file with `VRMLLoader`.

`THREE.BufferGeometry.computeBoundingSphere(): Computed radius is NaN. The "position" attribute is likely to have NaN values. `

This is VRML file. (the code is folded for readability)
screen shot 2018-09-03 at 16 42 41

### PR DESCRIPTION
**Pull request by Mugen87**
Fixes #14837.

### CODE REVIEW

File: examples/js/loaders/VRMLLoader.js [hunk 1]
```diff
@@ -1050,6 +1050,10 @@ THREE.VRMLLoader.prototype = {
 
 			var line = lines[ i ];
 
+			// The # symbol indicates that all subsequent text, until the end of the line is a comment,
+			// and should be ignored. (see http://www.agocg.ac.uk/train/vrml2rep/part1/guide3.htm)
```
Comment (WestLangley) [inline examples/js/loaders/VRMLLoader.js]: Is that link an authority?

I think the first line of your comment is adequate, unless comments are not in the official spec.

Comment (Mugen87) [reply examples/js/loaders/VRMLLoader.js]: I've used another link that directly refers to ISO/IEC 14772. This one is definitely trustworthy.

**Comment by mrdoob** [comment]
Thanks!

**Comment by Glinkis** [comment]
Doesn't this make this section obsolete, since it just seems to remove the whole comment from the parsed line?

```js
if ( /#/.exec( line ) ) {

	var parts = line.split( '#' );

	// discard everything after the #, it is a comment
	line = parts[ 0 ];

	// well, let's also keep the comment
	comment = parts[ 1 ];

}

if ( /#/.exec( line ) ) {

	var parts = line.split( '#' );

	// discard everything after the #, it is a comment
	line = parts[ 0 ];

	// well, let's also keep the comment
	comment = parts[ 1 ];

}
```

**Pull request merged**